### ISSUE TITLE
documentation of stable is broken

### ISSUE THREAD

**Issue by basnijholt** [python-adaptive/adaptive#157]
https://adaptive.readthedocs.io/en/stable/tutorial/tutorial.BalancingLearner.html

**Comment by basnijholt** [comment]
I don't think this can be fixed without releasing a new version.

**Comment by akhmerov** [comment]
We should also somehow set it to refuse to build whenever an error is raised.

**Comment by basnijholt** [comment]
> We should also somehow set it to refuse to build whenever an error is raised.

I am not sure if this is possible because in the documentation we raise an error somewhere to show how to debug.

I propose an alternative in https://github.com/python-adaptive/adaptive/pull/169.

**Comment by akhmerov** [comment]
We can incorporate this on the jupyter-sphinx side, see https://github.com/jupyter-widgets/jupyter-sphinx/issues/25

**Comment by basnijholt** [comment]
Yes, I agree that that would be a good solution, but not the full solution.

The reason why the docs were broken most recently (https://github.com/python-adaptive/adaptive/pull/168) wasn't because of an error but because of the fact that some javascript libs were loaded differently.

So I still think it is a good idea to be able to preview the docs somehow before merging.

**Comment by akhmerov** [comment]
Right, absolutely. Both are important.
